packages/@uppy/svelte/package.json

```diff
@@ -20,55 +44,37 @@
     "uppy-plugin",
     "svelte"
   ],
-  "files": [
```

Comment:
We should keep files around no?

https://github.com/transloadit/uppy/pull/5852

Reply:
if you remove it , would by default package * everything.